backoff.py

```diff
@@ -171,7 +171,7 @@ def emit(self, record):
 logger.setLevel(logging.ERROR)
 
 
-def expo(base=2, max_value=None):
+def expo(init_value=1, base=2, max_value=None):
```

Comment:
I still don't have a good idea what this param should be called, but I think init_value may be misleading. Isn't it more like a factor of current exponential value? Anyway, I won't block the PR on this, but I may end up changing this param name before a release.  (By the way, I would plan to release all this as backoff 1.2 once I'm sure we've got the API right.)